src/extends/room/meta.js

```diff
@@ -182,3 +182,11 @@ Room.isCuldesac = function (roomName, entrance) {
   // keeper room, or dead end appeared.
   return Object.keys(exits).length === 1
 }
+
+Room.isAvailable = function (name) {
+  if (Game.gcl.level > GCL_NOVICE) {
+    return Game.map.getRoomStatus(name).status === 'normal'
```

Comment:
What about respawn areas?

Reply:
If `isClaimable` gets thje GCL_NOVICE check would it be better to use simpler logic and just return `Game.map.getRoomStatus(name).status !== 'closed'`

Reply:
Yes, I think so. In the future we could add a method that checks whether a room is accessible based on whether it's in an area Quorum isn't in.